docker/archimedes-base/Dockerfile

```diff
@@ -38,21 +38,6 @@ RUN curl -o /tmp/dockerize.tgz -L $DOCKERIZE_URL && ( cd /usr/bin && tar xzf /tm
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py > /opt/install-poetry.py
 RUN python /opt/install-poetry.py --version $POETRY_VERSION
 
-# Install R from CRAN repo, to get latest
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-key '0xE19F5F87128899B192B1A2C2AD5F960A256A04AF'
```

Comment:
R is not needed in archimedes right now, and recently we discovered that this package source fails to install due to some recent changes upstream.  Rather than get that working again, let's remove it for now since it is not even used.